src/common/ZulipStatusBar.js

```diff
@@ -24,14 +24,14 @@ export const getStatusBarStyle = (statusBarColor: string): BarStyle =>
     ? 'light-content'
     : 'dark-content';
 
-type Props = {
+type Props = $ReadOnly<{
```

Comment:
nit: In the commit message:

    This is an automated transformation, performed by the following:
    
      # type Props = {| ... |};
      find src -name '*.js' | \
        xargs perl -i -0pe 's/(type Props \= )(\{\|.*?\|\})(;)/$1\$ReadOnly<$2>$3/
sg'
    
      # type Props = {| ... |};
      find src -name '*.js' | \
        xargs perl -i -0pe 's/(type Props \= )(\{.*?\})(;)/$1\$ReadOnly<$2>$3/sg'

I think the latter comment was meant to not have the `|` pair.